### PR TITLE
docs: add docstrings to functions in job_service.py

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## Issues related to this change
https://github.com/equinor/dm-docs/issues/35

### CODE REVIEW

File: src/services/job_service.py [hunk 1]
```diff
@@ -96,25 +112,58 @@ def _get_job(self, job_uid: UUID) -> Union[Job, None]:
...
+
+        - **document_id**: the address to the entity we want to update. Can be on the formats:
+          - By id: PROTOCOL://DATA SOURCE/$ID.Attribute
+          - By path: PROTOCOL://DATA SOURCE/ROOT PACKAGE/SUB PACKAGE/ENTITY.Attribute
+
+        - **reference**: an entity of type 'dmss://system/SIMOS/Reference' to be inserted.
+        """
         headers = {"Access-Key": token}
+        # TODO use document update instead, the reference insert endpoint has been removed from DMSS
         req = requests.put(f"{config.DMSS_API}/api/reference/{document_id}", json=reference, headers=headers)
```
Comment (ingeridhellen) [inline src/services/job_service.py:138]: Didn't you delete this endpoint?

File: src/services/job_service.py [hunk 2]
```diff
@@ -96,25 +112,58 @@ def _get_job(self, job_uid: UUID) -> Union[Job, None]:
...
+
+        - **job_entity**: the new job entity.
+        """
         return update_document_by_uid(dmss_id, {"data": job_entity}, token=token)
 
     def _get_job_handler(self, job: Job) -> JobHandlerInterface:
-        data_source_id = job.dmss_id.split("/", 1)[0]
+        """Get the job handler for a job.
+
+        Job handlers must be placed in the folders "default_job_handlers" and "job_handler_plugins" in the
```
Comment (ingeridhellen) [inline src/services/job_service.py]:
```suggestion
        Job handlers must be placed in the "job_handlers" folder in the
```
, right?

**Review by ingeridhellen — APPROVED** [review]

File: src/services/job_service.py (hunk 1 above)
Comment (KristianKjerstad) [reply src/services/job_service.py:138]: yes, that's why the TODO is added. Also added an issue for it. https://github.com/equinor/dm-job/issues/42

File: src/services/job_service.py (hunk 2 above)
Comment (KristianKjerstad) [reply src/services/job_service.py]: ups, it is actually "in the "default_job_handlers" or "job_handler_plugins" folder in the repository src folder."

File: src/services/job_service.py (hunk 1 above)
Comment (ingeridhellen) [reply src/services/job_service.py:138]: Haha, I didn't notice

**Pull request merged**